packages/tocco-ui/src/Preview/Preview.js

```diff
@@ -10,48 +10,60 @@ import {validateCssDimension} from '../utilStyles'
 /**
  * Use <Preview> to display a preview of any kind of file. Provide URLs to thumbnail and file.
  */
-const Preview = props => {
-  const onClick = () => {
-    if (typeof props.onClick === 'function') {
-      props.onClick(props.srcUrl, props.thumbnailUrl)
+class Preview extends React.Component {
+  state = {
+    loaded: false
+  }
+
+  handleOnClick = () => {
+    if (typeof this.props.onClick === 'function') {
```

Comment:
in my opinion we could shorten this to `if (this.props.onClick)`, since the prop type already checks if it's a function.

what do you think, @MethodenMann ?

Reply:
👍